.vib/mariadb/goss/goss.yaml

```diff
@@ -1,11 +1,10 @@
 gossfile:
   # Goss tests exclusive to the current container
-  ../../mariadb/goss/templates/mariadb.yaml: {}
+  ../../mariadb/goss/mariadb.yaml: {}
   # Load scripts from .vib/common/goss/templates
   ../../common/goss/templates/check-app-version.yaml: {}
+  ../../common/goss/templates/check-binaries.yaml: {}
   ../../common/goss/templates/check-broken-symlinks.yaml: {}
   ../../common/goss/templates/check-ca-certs.yaml: {}
-  ../../common/goss/templates/check-components-binaries.yaml: {}
-  ../../common/goss/templates/check-directories-permissions.yaml: {}
```

Comment:
You forgot to add a line with the new test for this `../../common/goss/templates/check-directories.yaml: {}`

Reply:
Thanks for the catch!